### ISSUE TITLE
ASP.NET Core 2.2/3.0 preview Jwt Bearer to API example (non Postman)

### ISSUE THREAD

**Issue by DanoThom** [aspnet/AspNetCore.Docs#13215]
TL/DR: while using Azure AD B2C flow we need a full web client to API example without using postman in the second step to grab the access token from the IdP.

As stated in the title, what would be immensely helpful would be just a single, clear example using a modern ASP.NET Core (I.E not half Postman) implementation of the most simple Web Client(UI) to Web Service (API) interaction while using the Azure AD B2C signup/signin flow.

Using the Postman example above is fine for testing but having a solid .NET Core code offering for actual real life use is much more valuable and much needed. I don't think that's too much to ask at this point as trying to use Azure AD B2C to get a usable jwt access token to pass on to a downstream API without an absolute mess of ADAL or MSAL and a hodgepodge of other tech just comes across very badly indeed.

I've lost countless hours trawling through the MS docs and various GitHub example projects and I'm extremely disappointed. The docs for B2C in general are extremely confusing and in a pretty poor, disheveled state with very little cohesion. The GitHub examples don't fare much better as they're either very out of date or far too simplified or just aren't applicable examples for common scenarios.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6ebb27a5-41cc-b64f-de68-2f0df6b02f8d
* Version Independent ID: 31de5338-ace5-b8c4-88e8-54863428fc61
* Content: [Authentication in web APIs with Azure Active Directory B2C in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/azure-ad-b2c-webapi?view=aspnetcore-3.0)
* Content Source: [aspnetcore/security/authentication/azure-ad-b2c-webapi.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/authentication/azure-ad-b2c-webapi.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @CamSoper
* Microsoft Alias: **casoper**

**Comment by softvitals** [comment]
Good

**Comment by tjopsta** [comment]
Are we any further with these examples?  Having the same as requested above for Blazor server side apps would be amazing.

**Comment by CamSoper** [comment]
@tjopsta I am actively working on updating docs in this area. ETA ~2-3 weeks.

**Comment by tjopsta** [comment]
Awesome, thanks very much for the quick update Cam!

**Comment by Rick-Anderson** [comment]
Moved to #16186
